src/components/SpaceProposalVotesList.vue

```diff
@@ -151,6 +151,15 @@ watch(visibleVotes, () => {
         <BaseButtonIcon @click="openReceiptModal(vote.ipfs)">
           <BaseIcon name="signature" />
         </BaseButtonIcon>
+        <span
+          v-if="vote.reason !== ''"
+          v-tippy="{
+            content: vote.reason
+          }"
+          class="text-skin-text transition-colors duration-200 hover:text-skin-link"
+        >
+          <BaseIcon name="info" />
```

Comment:
Hi @twodam, you don't need to add it, all hero icons are already installed and can be used like this.
```
<i-ho-annotation />
```
To change the size you can use tailwind css text size.

The `<BaseIcon>` is how we did it before and we haven't fully migrated all icons yet.

Reply:
Here is a full list of the installed hero icons https://icon-sets.iconify.design/heroicons-outline/